test/embind/test_i64_binding.cpp

```diff
@@ -13,79 +13,76 @@
 using namespace emscripten;
 using namespace std;
 
-#define assert_js(X) assert(run_js(X))
+#define assert_js_eq(X, Y) run_js(string("const x = ") + X + ", y = " + Y + "; assert(x === y, `" + X + ": actual = ${typeof x} ${x}, expected = ${typeof y} ${y}`);")
 
 void test(string message)
 {
```

Comment:
If you are gonna clean up this test maybe move all the open parens onto the declaration line?